apps/web/src/utils/cookies.ts

```diff
@@ -43,6 +43,6 @@ export function setNovuOnboardingStepCookie() {
   return novuOnboardedCookie.set('1', {
     expires: ONBOARDING_COOKIE_EXPIRY_DAYS,
     sameSite: 'none',
```

Comment:
There is one more type here:

```suggestion
    sameSite: 'None',
```